### DISCUSSION TITLE
In book theme, changing "in this article" to "table of contents" or "in this chapter"

### DISCUSSION THREAD

**Discussion by paulskrzypczyk** [jupyter-book/mystmd#1694] [Q&A] [answered]
Hi,

I am currently writing up some lecture notes using mystmd (which I think is fantastic!). I was wondering if there is any way to change the title of the right-hand outline. It currently always says "In this article" but this isn't really correct in my case, since each page of the site is either a chapter of the notes (or an associated set of problems). Ideally I would love to be able for this to say "Table of Contents" or "In this chapter". 

Is there a way to change this that I am missing? If not, can I request this as a small feature to be implemented? 

Many thanks
Paul

**Comment by rowanc1** [comment, marked as answer]
This is not currently possible in the theme, and as you note it is a small feature request. We have a developer sprint later this week, and I will suggest we tackle it!

**Reply by paulskrzypczyk** [reply]
Many thanks for the speed reply and positive response! I'll keep watch then!

**Reply by paulskrzypczyk** [reply]
Thanks for the update!

**Reply by agoose77** [reply]
This has been fixed in https://github.com/jupyter-book/myst-theme/pull/519!

**Reply by paulskrzypczyk** [reply]
I just updated myst and realised it had been updated, which prompted the thanks! (I hadn't realised there was a different thread on it)

Many thanks again!
Paul
